### DISCUSSION TITLE
Difference between RealmResourceProvider and AdminRealmResourceProvider

### DISCUSSION THREAD

**Discussion by blevine** [dasniko/keycloak-extensions-demo#20] [Q&A]
Hi,

In the rest-endpoint sample, I noticed that you use RealmResourceProvider for one endpoint and AdminRealmResourceProvider for the other. What is the difference between those two interfaces?

Thanks!

**Comment by dasniko** [comment]
The one is for realm resources (endpoints unter regular realm paths), the other one for admin realm resources (endpoints under the admin path / admin api).
